recipes/LibriSpeech/LM/extra_requirements.txt

```diff
@@ -1,2 +1,2 @@
 # huggingface dataset
-datasets==1.6.2
+datasets
```

Comment:
I am all about using frozen libs thought :p

Reply:
fair enough - frozen libs need to be checked & addressed with SB major version preparations though. It would be good to have a more explicit comment of why this freeze is. Freezing everything once created will fragment extra dependencies within a major version cycle. That is fine for that cycle (e.g. all that is in 0.5.*); yet, for just before a next major version (e.g. 0.6.0), these dependency issues should be revisited—to keep up with the larger community which also moves on (as we do also).